tutorials/Publish_NeMo_Model_On_Hugging_Face_Hub.ipynb

```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "Publish_NeMo_Model_On_Hugging_Face_Hub.ipynb",
+      "name": "Copy of Publish_NeMo_Model_On_Hugging_Face_Hub.ipynb",
```

Comment:
Oh no, @ericharper can you rename this to just Publish... and remove the Copy of part? My bad..